plugins/modules/source_control/gitlab/gitlab_service.py

```diff
@@ -0,0 +1,467 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Raphaël Droz <raphael@droz.eu>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: gitlab_service
+short_description: Creates, updates or deletes GitLab services (aka "integrations").
+version_added: '1.3.0'
+description:
+  - Setup or delete GitLab integration services.
+author:
+  - Raphaël Droz (@drzraf)
+requirements:
+  - python >= 2.6
+  - python-gitlab python module
+extends_documentation_fragment:
+- community.general.auth_basic
+
+options:
+  project:
+    description:
+      - The ID or path of the project (urlencoded or not)
+    required: true
+    type: str
+  active:
+    description:
+      - Not yet supported (U(https://gitlab.com/gitlab-org/gitlab-ce/issues/41113))
```

Comment:
```suggestion
      - Not yet supported (U(https://gitlab.com/gitlab-org/gitlab-ce/issues/41113)).
```
Always make paragraphs (list elements) in description proper sentences. I won't comment for the others.